Bootstrap.php

```diff
@@ -96,13 +97,19 @@ public function bootstrap($app)
 
             if (!isset($app->get('i18n')->translations['user*'])) {
                 $app->get('i18n')->translations['user*'] = [
-                    'class'    => PhpMessageSource::className(),
+                    'class' => PhpMessageSource::className(),
                     'basePath' => __DIR__ . '/messages',
                     'sourceLanguage' => 'en-US'
                 ];
             }
 
             Yii::$container->set('dektrium\user\Mailer', $module->mailer);
+
+            // Ensure the module is not in DEBUG mode on production environments
+            if (!defined('YII_DEBUG') || !defined('YII_ENV')
```

Comment:
I don't like this multi-line `if` :smile: 
Maybe it's better to move it to new method called `isDebugEnabled`, and use here:
```php
if (!$this->isDebugEnabled() {
    $module->debug = false;
}
```